src/index.ts

```diff
@@ -13,14 +13,15 @@ import { selectMethod } from './util/selectMethod';
  * @param options - Options.
  * @returns - An object with fitting error and the list of optimized parameters { parameters: [ {x, y, width} ], error } if the kind of shape is pseudoVoigt mu parameter is optimized.
  */
+
 export function optimize(
   data: DataXY<DoubleArray>,
-  peakList: Peak1D[],
+  peakListInitial: Peak1D[],
   options: {
     /**
-     * kind of shape used for fitting
-     **/
-    shape?: Shape1D;
+     * the default peak shape if not specified
+     */
+    defaultPeakShape?: string;
```

Comment:
We should keep a Shape1D kind because this default shape may have parameters (like pseudovoigt).

I agree that the defaultPeakShape is better but I would still keep the name 'shape' because most of the time this code will be used with only one shape.